Pyjion/absint.cpp

```diff
@@ -1616,6 +1616,39 @@ void AbstractInterpreter::build_set(size_t argCnt) {
     inc_stack();
 }
 
+void AbstractInterpreter::extend_set_recursively(Local setTmp, size_t argCnt) {
+    if (argCnt == 0) {
+        return;
+    }
+
+    auto valueTmp = m_comp->emit_define_local();
```

Comment:
I just copied what I did for list extensions and tweaked some method names, so it happens to be how we solved this previously. I vaguely remember debugging some issue that led to this approach.

How about we merge this and then I fix both this and lists?